jpa/hibernate-transformer/src/main/java/org/jboss/as/hibernate/MethodAdapter.java

```diff
@@ -78,9 +78,6 @@ public void visitMethodInsn(int opcode, String owner, String name, String desc,
                 owner.equals("org/hibernate/Query") &&
                 name.equals("getFirstResult") &&
                 desc.equals("()Ljava/lang/Integer;")) {
-            logger.debugf("Deprecated Hibernate51CompatibilityTransformer transformed application classes in '%s', " +
-                            "class '%s' is calling %s.%s, which must be changed to expect int result, instead of Integer.",
-                    moduleName, className, name, owner);
             logger.debugf("Deprecated Hibernate51CompatibilityTransformer transformed application classes in '%s', " +
                             "class '%s', is calling org.hibernate.Query.getFirstResult, which must be changed to call getHibernateFirstResult() " +
```

Comment:
`BasicQueryContract#getHibernateFirstResult` was introduced in 5.3 as a deprecated method, and was added as a temporary measure. Applications should be changed to use the JPA methods (`javax.persistence.Query#getFirstResult` and #setFirstResult), and ensure that their usage complies with the JPA specification.

Reply:
Should we revert the deletion of the DEBUG logged message?  Or add a new DEBUG logged message with, your comment as the text?

Reply:
This also applies to logging for org.hibernate.Query#setFirstResult

Reply:
Sorry, I was misreading the log messages. I see they are just documenting what the transformer has transformed. They do not provide recommendations for actual changes to the application. 

I like that the log messages start with "Deprecated Hibernate51CompatibilityTransformer". It makes it clear that it is a temporary measure.

I think it would be a good idea to put "deprecated" before "getHibernateFirstResult", "setHibernateFirstResult", "getHibernateMaxResults", and "setHibernateMaxResults" in the log messages as well to give the same idea.

Reply:
I just want to mention that `getHibernateFlushMode` is *not* deprecated.

Reply:
Hi Gail, I think that when users start actually migrating their application and write code that tries to call the deprecated Hibernate ORM methods, they should see warnings in their IDE and compiler warnings as well.  

The Hibernate51CompatibilityTransformer is different, as users will never write code to that users the Hibernate51CompatibilityTransformer directly, instead if they enable it, either they will read in our doc, that it is deprecated or they might see the DEBUG message that mentions that Hibernate51CompatibilityTransformer is deprecated but they won't get warnings in their IDE or from compiling their application code.